wordpress/wp-content/themes/headless/setup/helpers/stellate.php

```diff
@@ -0,0 +1,145 @@
+<?php
+// Disable a plugin based on environment. We don't want to purge cache during local/staging development.
+// Also allows for listening to graphcdn_purge event to perform additional invalidation
+
+// add_action('admin_init', 'disable_stellate_plugin');
+
+// TODO: we want want our staging/dev environments hitting production, so we need to override the service name and token
```

Comment:
I'm a litle confused on this one, it's a PHP file, so are we saying we want our staging/dev wordpress to hit production stellate?

Reply:
this was a leftover comment from elsewhere, removed